test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/CS/Source/NullPropagationExpressions.cshtml

```diff
@@ -0,0 +1,13 @@
+﻿@{
+    @ViewBag?.Data
+    @ViewBag.Method?(Hello World)
```

Comment:
Oh man I thought that Method?() was supported for delegates. Just tried it out, I be wrong, will remove.